website/source/index.html.erb

```diff
@@ -1,144 +1,434 @@
+        <h2>Simplify Operations</h2>
+        <p class="lead">
+          Nomad simplifies operations by supporting blue/green deployments,
+          automatically handling machine failures, and providing a single
+          workflow to deploy applications.
+        </p>
+      </div>
+    </div>
 
-        <div id="deploy" class="feature">
-          <div class="feature-header">
-            <h3>Easily Deploy</h3>
-            <p>
-              Users submit simple high-level jobs and Nomad handles
-              scheduling, deploying and upgrading applications.
-            </p>
-          </div>
-          <div class="feature-graphic"></div>
-          <div class="feature-footer">
-            <p>Nomad makes it easy to deploy one container or thousands.</p>
-            <span class="docker-outline-logo"></span>
-          </div>
-        </div>
+    <div class="row">
+      <div class="col-sm-offset-5 col-sm-6">
+        <h3>Rolling Deploys</h3>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-sm-5" id="feature-rolling-deploys">
+        <%= inline_svg "feature-rolling-deploys.svg" %>
+      </div>
+      <div class="col-sm-7">
+        <p>
+          In order to update an application while reducing downtime, Nomad
+          provides a built-in mechanism for rolling upgrades. Operators specify
+          their update strategy in the declarative job file, plan the changes,
+          and submit them for evaluation and allocation. With full control over
+          parallelization and stagger, rolling upgrades make application
+          deployments simple and safe.
+        </p>
+      </div>
+    </div>
 
-        <div id="partners" class="feature">
-          <div class="feature-header right clearfix">
-            <div class="row">
-              <h3 class="">Any Cloud or Every Cloud</h3>
-            </div>
-            <div class="row">
-              <p>
-                Multi-Datacenter and Multi-Region support enables running applications
-                on any cloud, public or private.
-              </p>
-            </div>
-          </div>
-          <div class="partner-logos">
-            <div class="row">
-              <div class="col-sm-2 docker-wrap">
-                <span class="partner-logo docker-logo"></span>
-              </div>
-              <div class="col-sm-10 cloud-wrap clearfix">
-                <span class="partner-logo aws-logo"></span>
-                <span class="partner-logo google-logo"></span>
-                <span class="partner-logo microsoft-logo"></span>
-                <span class="partner-logo digitalocean-logo"></span>
-              </div>
-            </div>
-          </div>
-        </div>
+    <div class="row">
+      <div class="col-sm-12">
+        <h3>Blue/Green Deployments</h3>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-sm-7">
+        <p>
+          Nomad supports blue/green and canary deployments through the
```

Comment:
This is actually first classed now: https://github.com/hashicorp/nomad/blob/master/website/source/docs/job-specification/update.html.md#bluegreen-upgrades